catalog/forms.py

```diff
@@ -0,0 +1,64 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+from catalog.models import Cook, Dish
+
+
+class CookExperienceUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Cook
+        fields = ["years_of_experience"]
+
+    def clean_years_of_experience(self):
+        return validate_years_of_experience(
+            self.cleaned_data["years_of_experience"])
+
+
+def validate_years_of_experience(value):
+    if value < 0:
+        raise forms.ValidationError("Years cannot be negative")
+    return value
```

Comment:
Nie widzę nigdzie indziej użycia funkcji `validate_years_of_experience`, a ponadto jej logika jest na tyle prosta, ze mogłaby być od razu częścią `clean_years_of_experience`

```suggestion
    def clean_years_of_experience(self):
        value = self.cleaned_data["years_of_experience"]
        if value < 0:
            raise forms.ValidationError("Years cannot be negative")
        return value
```

Reply:
Zmieniłem walidację na tą w modelu.